main.js

```diff
@@ -0,0 +1,122 @@
+    image: 'assets/images/img4.svg',
+    languages: ['HTML', 'CSS', 'JavaScript'],
+    technologies: ['HTML', 'CSS', 'Ruby on rails', 'JavaScript', 'GitHub'],
+    seeLive: 'https://bludidi.github.io/Portfolio-Desktop-version-/',
+    seeSource: 'https://github.com/Bludidi/Portfolio-Desktop-version-',
+  },
+];
+
+const projectCards = projects.map((project) => `
+<div id="card1" class="cards">
+            <div id="card_img" class="card left_align"><img class="card_img" src="${project.image}" alt="project card"></div>
+              <div id="txt_img" class="img-txt1"><h2>${project.name}</h2>
+                <ul class="flex-cards">
+                  <li class="all-list first">${project.subtitle}</li>
+                  <li class="all-list second">${project.backEnd}</li>
+                  <li class="all-list second">${project.year}</li>
+                </ul>
+                <p class="p3">${project.shortDescription}</p>
+                <ul class="flex-button tech_lang">
+                ${project.languages.map((tech) => `<li class="button">${tech}</li>`).join('')}
+                </ul>
+                <br>
+                <input class="btns see_btn" type="button" value="See Project">
+            </div>
+        </div>
+`);
+
+const modalContainer = document.querySelector('.modal');
+const recentWork = document.querySelector('.myWork');
+const parser = new DOMParser();
+projectCards.forEach((projectCard, index) => {
+  const projectIndex = parser.parseFromString(projectCard, 'text/html').body.firstChild;
+  const popupModal = `
+<div class="project-popup-hidden ">
+  <div class="backgroundpop"></div>
+    <div class="project-popup">
+      <div class="project-header">
+        <input type="button" value="&#10006" class="fa-solid fa-xmark project-close close">
+        <h2>${projects[index].name}</h2>
+        <p class="all-list first">${projects[index].subtitle} <span class="all-list second">${projects[index].backEnd}</span> <span class="all-list second">${projects[index].year}</span></p>
+
+        <div class="card_img card1">
+          <img class="imagedesktop project-img"  src="${projects[index].image}" alt="card image">            
+          <div class="project-footer">
+          <p class="project-desc">${projects[index].description}</p>
+        <div>
+          <ul class = ("tech_lang")>
+            ${projects[index].technologies.map((tech) => `<li>${tech}</li>`).join('')}
+          </ul>
+
+          <div class="project-links">
+            <a href="${projects[index].seeLive}">See Live <i class="close"></i></a>
+            <a href="${projects[index].seeSource}">See Source <i class="close"></i></a>
```

Comment:
- [ ] Kindly follow design for Desktop and  mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, text, margins). Please try to add the icon for "See Live" and " See Source" based on the Figma design.
The Figma design Look likes:
![Capture1](https://user-images.githubusercontent.com/93598189/190616025-21b8a213-0a52-40fc-88cd-0421a0cb6145.PNG)

Your Design Look Likes This  : 
![Capture](https://user-images.githubusercontent.com/93598189/190616098-54d9e2d1-b93f-4a04-a3d2-56982756f17d.PNG)